Recipely/Recipely/Modules/RecipesModule/CategoryDishesScreen/View/CategoryDishesView.swift

```diff
@@ -165,73 +151,34 @@ class CategoryDishesView: UIViewController, UIGestureRecognizerDelegate {
-        case .notPressed:
-            timeView.backgroundColor = .recipeView
-            timeView.changeParameters(title: Constants.timeText, image: .stackIcon)
-        case .sortingMore:
-            timeView.backgroundColor = .accent
-            timeView.changeParameters(title: Constants.timeText, image: .stackIcon.withTintColor(.white))
-        case .sortingSmaller:
-            guard let image = UIImage.stackIcon.cgImage else { return }
-            let newImage = UIImage(cgImage: image, scale: 1, orientation: .downMirrored).withTintColor(.white)
-            timeView.backgroundColor = .accent
-            timeView.changeParameters(title: Constants.timeText, image: newImage)
-        }
-    }
-
-    func changesCaloriesSortingStatus(condition: Condition) {
-        switch condition {
-        case .notPressed:
-            caloriesView.backgroundColor = .recipeView
-            caloriesView.changeParameters(title: Constants.caloriesText, image: .stackIcon)
-        case .sortingMore:
-            caloriesView.backgroundColor = .accent
-            caloriesView.changeParameters(title: Constants.caloriesText, image: .stackIcon.withTintColor(.white))
-        case .sortingSmaller:
-            guard let image = UIImage.stackIcon.cgImage else { return }
-            let newImage = UIImage(cgImage: image, scale: 1, orientation: .downMirrored).withTintColor(.white)
-            caloriesView.backgroundColor = .accent
-            caloriesView.changeParameters(title: Constants.caloriesText, image: newImage)
-        }
+    func reloadDishes() {
+        tableView.reloadData()
     }
 }
 
 extension CategoryDishesView: UITableViewDataSource {
-    func numberOfSections(in tableView: UITableView) -> Int {
-        content.count
-    }
-
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        presenter?.dishes.count ?? 0
+        presenter?.getNumberDishes() ?? 0
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let items = content[indexPath.section]
-        switch items {
-        case .basicDishCell:
-            guard let category = presenter?.dishes,
-                  let cell = tableView.dequeueReusableCell(
-                      withIdentifier: BasicDishCell.description(),
-                      for: indexPath
-                  ) as? BasicDishCell
-            else { return UITableViewCell() }
-            cell.configureCell(category: category[indexPath.row])
+        guard let dish = presenter?.getDish(forIndex: indexPath.row) else { return UITableViewCell() }
+        switch dish {
+        case let .data(dish):
+            guard let cell = tableView.dequeueReusableCell(
+                withIdentifier: DishCell.description(),
+                for: indexPath
+            ) as? DishCell else { return UITableViewCell() }
+            cell.configure(with: dish)
+            return cell
+        case .noData:
```

Comment:
```suggestion
        case .noData:
```
лучше .loading так как на этот кейс когда нет данных